types/Cms.ts

```diff
@@ -1,7 +1,13 @@
-// TODO: Add real type
-export type CreateArgs = {
-  branch?: string;
-};
+import { CommonArgs, ConfigArgs } from './Yargs';
+
+export type CreateArgs = CommonArgs &
```

Comment:
nit, this can live in the command file if it's only used there

Reply:
It's used in this file too!